api/admin/manual-ticket-entry.js

```
@@ -0,0 +1,352 @@
+    /\$\{.*\}/,
+    /__proto__/,
+    /constructor/,
+    /prototype/,
+    /eval\s*\(/i,
+    /function\s*\(/i,
+    /\.\.\//,
+    /union\s+select/i,
+    /insert\s+into/i,
+    /delete\s+from/i,
+    /drop\s+table/i,
+    /\x00|\x08|\x0B|\x0C/
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)